librz/util/sdb/src/sdb_private.h

```diff
@@ -35,6 +52,11 @@ static inline void ut32_pack(char s[4], ut32 u) {
 	s[3] = u >> 8;
 }
 
+/**
```

Comment:
In fact, these functiona should probably removed in favor of functions from `librz/include/rz_endian.h`
Now, that SDB is inlined, we can reuse functions from other parts of RzUtil whenever possible.

Reply:
Do I just include `<rz_endian.h>`? See new commit.

Reply:
Yes.